### PR TITLE
Bump dependencies.

### PR DESCRIPTION
**Pull request by michael-simons**
Some housekeeping before a possible next beta.

I bumped all of them in one go @meistermeier

### CODE REVIEW

File: pom.xml [hunk 1]
```diff
@@ -82,30 +82,31 @@
 
 	<properties>
 		<apiguardian.version>1.0.0</apiguardian.version>
-		<assertj.version>3.11.1</assertj.version>
+		<byte-buddy.version>1.9.10</byte-buddy.version>
+		<assertj.version>3.15.0</assertj.version>
```
Comment (meistermeier) [inline pom.xml]: Flip those lines ;)

File: pom.xml [hunk 2]
```diff
@@ -128,6 +129,16 @@
 
 	<dependencyManagement>
 		<dependencies>
+			<dependency>
```
Comment (meistermeier) [inline pom.xml]: Why do we have a direct dependency on bytebuddy now?

Comment (michael-simons) [reply pom.xml]: Non-converging otherwise.

File: pom.xml (hunk 1 above)
Comment (michael-simons) [reply pom.xml]: 🤦‍♂ Yeah.

**Pull request merged**